docker-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if test -f /var/www/html/inc/conf.php; then
+  if chown www-data:www-data /var/www/html/inc/conf.php ; then
+    echo "conf.php was chown'ed properly"
+  else
+    echo "There were some errors while chown'ing conf.php"
```

Comment:
adding an exit error, so container fails out whenever there was an error.

```suggestion
    echo "There were some errors while chown'ing conf.php"
    exit 1
```